spec/rails/class_loading_system_spec.rb

```diff
@@ -33,11 +36,29 @@
   end
 
   it "should not allow me to define widgets outside of app/views/" do
-    expect_exception('widget_defined_outside_app_views', 'ActionView::MissingTemplate', /class_loading_system_spec\/widget_defined_outside_app_views/)
+    # TODO: what is this spec testing? the action doesn't tell rails to render the outside widget
```

Comment:
yeah, that seems to be it. the controller action for this spec loads in a rails file that defines `Views::ClassLoadingSystemSpec::WidgetDefinedOutsideAppViews`. This spec is testing to make sure that implicit rendering does not render that widget, since demands that all templates are under app/views